chutney/ui/src/app/modules/campaign/components/execution/history/campaign-executions-history.component.ts

```diff
@@ -85,6 +86,7 @@ export class CampaignExecutionsHistoryComponent implements OnInit, OnDestroy {
         return this.campaignService.find(this.campaignId).pipe(
             tap(campaign => this.campaign = campaign),
             tap(campaign => this.campaignReports = campaign.campaignExecutionReports.map(cer => new CampaignReport(cer))),
+            tap(campaign => this.canReplay = campaign.campaignExecutionReports.length > 0),
```

Comment:
May be check if status is not running or not_executed instead ?

Reply:
Campain does not have the information of the status, i only have the reports
https://github.com/chutney-testing/chutney/blob/main/chutney/ui/src/app/core/model/campaign/campaign.model.ts#L19